packages/errors/src/index.ts

```diff
@@ -131,7 +132,9 @@ export const FileNotFound = createErrorType('FileNotFound');
 
 export const FTPError = createErrorType('FTPError');
 
-export const ApiCollisionError = createErrorType('ApiCollisionError');
+export const GranuleNotPublished = createErrorType('GranuleNotPublished');
+
+export const GranuleFileWriteError = createErrorType('GranuleFileWriteError');
```

Comment:
This change just adds a named error, the rest is an alpha sort for organization.